templates/cluster_role.yaml

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: openshift-config-collector
```

Comment:
Shouldn't the name include `insights` or something like that (same for the binding) ? Not sure...just asking :-)

Reply:
Wasn't sure it made senes for that naming to be applied here. I kept it generic as it seemed strange to me that Red Hat Marketplace or Cost Operator would create an insights named role/binding, but I'm not super opinionated about it. @jhjaggars Thoughts?

Reply:
I'm -0 on putting insights in the name.  The only reason I'd do that now is if we expect to collide with another name (logically) and it might cause confusion.

Reply:
We could add a OPERATOR_PREFIX to the template that gets updated so that we always avoid a name collision.

Reply:
+1 to some operator prefix in the name.

Reply:
Ok. Prefix option added in latest commit.